client/js/screens/SignUp/SignUpContainer.js

```diff
@@ -1,10 +1,41 @@
 import React, { Component } from "react";
 import Signup from "./Signup";
+import gql from "graphql-tag";
+import { Mutation } from "react-apollo";
+import { View } from "react-native";
 
-class SignupContainer extends Component {
-    render() {
-        return <Signup />;
+const SignupMutation = gql`
+  mutation CreateUser(
+    $email: String!
+    $password: String!
+    $firstname: String!
+    $lastname: String!
+  ) {
+    createUser(
+      email: $email
+      password: $password
+      firstname: $firstname
+      lastname: $lastname
+    ) {
+      id
+      email
     }
+  }
+`;
+
+class SignupContainer extends Component {
+  render() {
+    return (
+      <Mutation mutation={SignupMutation}>
```

Comment:
how are you passing variables in your Mutation query? Missing variables?

Reply:
the variable are on the signup.js cause the variables are the textInput